kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs

```diff
@@ -607,12 +607,6 @@ impl<'tcx> GotocCtx<'tcx> {
             }
             "unchecked_sub" => codegen_op_with_overflow_check!(sub_overflow),
             "unlikely" => self.codegen_expr_to_place(p, fargs.remove(0)),
-            "unreachable" => self.codegen_assert(
```

Comment:
Since `unreachable` should be unreachable, I suggest making this more explicit:
```rust
"unreachable" => unreachable!("Expecting unreachable intrinsic to be handled using Terminator::Unreachable")
```
because just removing it gives the impression that it is not yet implemented (gets included in the `_` bucket at the end of match statement).

Reply:
Ok, I will add this in a follow-up PR.